macros/plugin.py

```diff
@@ -17,8 +17,10 @@
 
 from mkdocs.plugins import BasePlugin
 from mkdocs.config.config_options import Type as PluginType
-from mkdocs.utils import string_types
-
+try:
```

Comment:
Put a short comment to explain issue, for our future selves?

Reply:
Done.

For future releases I would consider dropping the string_type completely, as python 2.7+ has already reached its end-of-life and is out of maintenance.

Reply:
Noted.